templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -74,6 +77,10 @@ class _Result(BaseModel):
     nodes: List[_SourceNodes]
 
 
+class _ChatConfig(BaseModel):
+    starterQuestions: Optional[List[str]]
+
```

Comment:
In python, we use snake case instead of camel case. You can try advance feature from pydantic to alias the field name and add the field description too.
```suggestion
class _ChatConfig(BaseModel):
    starter_questions: Optional[List[str]] = Field(
        default=None,
        description="List of starter questions",
    )

    class Config:
        json_schema_extra = {
            "example": {
                "starterQuestions": [
                    "What standards for letters exist?",
                    "What are the requirements for a letter to be considered a letter?",
                ]
            }
        }
        alias_generator = to_camel
```
and don't forget to update the import:
```python
from pydantic import BaseModel, Field
from pydantic.alias_generators import to_camel
```